src/mqttcm_connect.c

```diff
@@ -1586,14 +1673,18 @@ int mqtt_subscribe(char *comp, char *topic)
 
 		if(AddToSubscriptionList(comp ,topic))
 		{
-			rc = mosquitto_subscribe(mosq, NULL, topic, 1);
+			//Adding int pointer subscribId in mosquitto_subscribe function to get the unique message id for subscribe callback usage
```

Comment:
Modify this description to something like unique subscribeId will be sent from cloud after subscription of each component

Reply:
Also print the subscribed id just aft mosquitto subscribe to know what cloud sends and also print it after updating to list along with topic and component